content/en/docs/setup/getting-started/index.md

```diff
@@ -43,8 +43,8 @@ Follow these steps to get started with Istio:
     {{< tip >}}
     The command above downloads the latest release (numerically) of Istio.
     To download a specific version, you can add a variable on the command line.
-    For example to download Istio 1.4.3, you would run
-      `curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.4.3 sh -`
+    For example to download Istio 1.6.3, you would run
```

Comment:
Why not use _istio_full_version_ here as well?

Reply:
Because the instructions say

> To download a specific version, you can add a variable on the command line.

`istio_full_version` adds the latest release version. and as per the instruction, I think it means to download older version?

Reply:
It's probably fine to not update this at all, since it's just an example of some "other" release, so 1.4.3 is as good as any for the example.

Reply:
Done